packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1336,3 +1403,202 @@ class RenderMetaData extends RenderProxyBox {
   /// Opaque meta data ignored by the render tree
   dynamic metaData;
 }
+
+class RenderSemanticsGestureHandler extends RenderProxyBox implements SemanticActionHandler {
+  RenderSemanticsGestureHandler({
+    RenderBox child,
+    GestureTapCallback onTap,
+    GestureLongPressCallback onLongPress,
+    GestureDragUpdateCallback onHorizontalDragUpdate,
+    GestureDragUpdateCallback onVerticalDragUpdate,
+    this.scrollFactor: 0.8
```

Comment:
So that when you scroll you still see some of the previous page. This is completely arbitrary though. Would you prefer 1.0? I expect to tweak this based on real user feedback eventually.